bugbug/bugzilla.py

```diff
@@ -137,12 +137,14 @@ def download_bugs(bug_ids, products=None, security=False):
     new_bug_ids = sorted(list(new_bug_ids))
 
     total_downloaded = 0
+    pbar = tqdm(total=len(new_bug_ids))
     chunks = (new_bug_ids[i:(i + 500)] for i in range(0, len(new_bug_ids), 500))
+
     for chunk in chunks:
         new_bugs = _download(chunk)
 
         total_downloaded += len(new_bugs)
-
+        pbar.update(total_downloaded)
```

Comment:
I guess you haven't actually tested this?
The parameter to `update` is not the current total, but the step.

Reply:
Hey I modified the code and updated by length of chunk.
Is it fine now?